
From: Jesper Juhl <juhl-lkml@dif.dk>

Here's a small patch that lists X.Org as well as XFree86 in
Documentation/SubmittingDrivers in the section talking about video drivers.

Signed-off-by: Jesper Juhl <juhl-lkml@dif.dk>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Documentation/SubmittingDrivers |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN Documentation/SubmittingDrivers~tiny-update-to-documentation-submittingdrivers-list-xorg Documentation/SubmittingDrivers
--- 25/Documentation/SubmittingDrivers~tiny-update-to-documentation-submittingdrivers-list-xorg	2004-06-21 23:17:13.139372384 -0700
+++ 25-akpm/Documentation/SubmittingDrivers	2004-06-21 23:17:13.143371776 -0700
@@ -3,7 +3,8 @@ Submitting Drivers For The Linux Kernel
 
 This document is intended to explain how to submit device drivers to the
 various kernel trees. Note that if you are interested in video card drivers
-you should probably talk to XFree86 (http://www.xfree86.org) instead.
+you should probably talk to XFree86 (http://www.xfree86.org/) and/or X.Org
+(http://x.org/) instead.
 
 Also read the Documentation/SubmittingPatches document.
 
_
